pkg/resourcegenerator/maskinporten/maskinporten.go

```diff
@@ -23,7 +24,7 @@ func Generate(r reconciliation.Reconciliation) error {
 	}
 
 	if !MaskinportenSpecifiedInSpec(application.Spec.Maskinporten) {
-		ctxLog.Info("Maskinporten not specified in spec, skipping generation")
+		ctxLog.Debug("Maskinporten not specified in spec, skipping generation")
```

Comment:
Unrelated change, but changed to debug to follow every other generator so it can be filtered out if necessary